[Docs]: Update listed Node versions to v24

### I confirm that:

- [x] This issue is about the **documentation website**, NOT about the Navidrome server

### Page URL or path

https://www.navidrome.org/docs/developers/dev-environment/

### What's wrong

The [dev-environment](https://www.navidrome.org/docs/developers/dev-environment/) and [build-from-source](https://www.navidrome.org/docs/installation/build-from-source/) pages both reference `Node 20` (last updated in April 2024, as per 444459e64c06c6e5a2cfd2a845246f4c85cb791b and c31feefffe75a34aec44e6874b3128a01f6e98f6), even though Navidrome now explicitly requires `v24` (as of [October 2025](https://github.com/navidrome/navidrome/pull/4618)).

This led me to the following error when running `make setup`:

> ERROR: Please check your Node version. Should be at least v24
>
> make: *** [check_node_env] Error 1

### Suggested fix

To update the dev-environment](https://www.navidrome.org/docs/developers/dev-environment/) and [build-from-source](https://www.navidrome.org/docs/installation/build-from-source/) pages to reference `Node 24` instead.

--- a/content/en/docs/developers/dev-environment.md
+++ b/content/en/docs/developers/dev-environment.md
@@ -23,7 +23,7 @@ Keep in mind that the overall experience when using Docker Desktop for developme
 ### Unix-based systems (Linux, macOS, BSD, …)
 
 1. Install [GoLang 1.23+](https://golang.org/doc/install)
-2. Install [Node 20](http://nodejs.org/)
+2. Install [Node 24](http://nodejs.org/)
 3. Install [TagLib 2.0+](https://github.com/taglib/taglib/blob/master/INSTALL.md)
     - Arch Linux: `pacman -S taglib`
     - macOS: `brew install taglib --HEAD`

--- a/content/en/docs/installation/build-from-source.md
+++ b/content/en/docs/installation/build-from-source.md
@@ -19,7 +19,7 @@ you should open an [issue in the project's GitHub page](https://github.com/navid
 If you don't want to wait, you can try to build the binary yourself, with the following steps.
 
 First, you will need to install [Go 1.24+](https://golang.org/doc/install) and
-[Node 20+](https://nodejs.org/en/download). The setup is very strict, and the steps below only work with
+[Node 24+](https://nodejs.org/en/download). The setup is very strict, and the steps below only work with
 these versions (enforced in the Makefile). Make sure to add `$GOPATH/bin` to your `PATH` as described
 in the [official Go site](https://golang.org/doc/gopath_code.html#GOPATH)
 

